src/pages/lib-v1/auth/getting-started/q/platform/[platform].mdx

```diff
@@ -26,3 +26,257 @@ import android2 from '/src/fragments/lib-v1/auth/native_common/getting_started/c
+- Enable Lambda functions to access the chosen Cognito resource if you permit it
+
+> If you have previously enabled an Amplify category that uses Auth behind the scenes (e.g. API category), you can run the `amplify update auth` command to edit your configuration if needed. 
+
+After configuring your Authentication options, update your backend and deploy the service by running the `push` command:
+
+```bash
+amplify push
+```
+
+Now, the authentication service has been deployed and you can start using it. To view the deployed services in your project at any time, go to Amplify Console by running the following command:
+
+```bash
+amplify console
+```
+
+In your app's entry point (i.e. **App.js**, **index.js**, **_app.js**, or **main.js**), import and load the configuration file:
+
+```javascript
+import { Amplify } from 'aws-amplify';
+import awsconfig from './aws-exports';
+
+Amplify.configure(awsconfig);
+```
+
+</Block>
+
+<Block name="Existing Resources">
+
+Existing Authentication resources from AWS (e.g. Amazon Cognito UserPools or Identity Pools) can be used with the Amplify Libraries by calling the `Amplify.configure()` method. 
+
+In your app's entry point (i.e. **App.js**, **index.js**, **_app.js**, or **main.js**), import and load the configuration file:
+
+```javascript
+import { Amplify } from 'aws-amplify';
+
+Amplify.configure({
+  Auth: {
+    // Configure Amplify for use with your user pool
+    Cognito: {
+      // OPTIONAL - Amazon Cognito User Pool ID
+      userPoolId: 'XX-XXXX-X_abcd1234',
+
+      // OPTIONAL - Amazon Cognito Web Client ID (26-char alphanumeric string)
+      userPoolClientId: 'a1b2c3d4e5f6g7h8i9j0k1l2m3',
+
+      // OPTIONAL - This is used when autoSignIn is enabled for Auth.signUp
+      // 'code' is used for Auth.confirmSignUp, 'link' is used for email link verification
+      signUpVerificationMethod: 'code', // 'code' | 'link'
+
+      // OPTIONAL - Hosted UI configuration
+      loginWith: {
+        oauth: {
+          domain: 'your_cognito_domain',
+          scope: [
+            'phone',
+            'email',
+            'profile',
+            'openid',
+            'aws.cognito.signin.user.admin',
+          ],
+          redirectSignIn: ['http://localhost:3000/'],
+          redirectSignOut: ['http://localhost:3000/'],
+          responseType: 'code', // or 'token', note that REFRESH token will only be generated when the responseType is code
+        },
+      }
+    }
+  },
+});
+
+// Get the current Amplify configuration
+const currentConfig = Amplify.getConfig();
+```
+<Callout warning={true}>
+If your existing UserPool client has a required attribute that is NOT set to mutable, you may face login issues when using Social sign-in. To resolve this, you will need to <ExternalLink href="https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-settings-client-apps.html">create a new UserPool client</ExternalLink> and mark the required attribute as mutable.
```

Comment:
The writing of "user pool" is inconsistent across many pages. I believe Cognito's guidance is "Cognito user pool" on first occurrence, then subsequent occurrences can just be "user pool". Suggest a find + replace.

Reply:
Standardized on `Cognito user pool` for first occurrence, and `user pool` for subsequent occurrences.